content/docs/user-guide/external-dependencies.md

```diff
@@ -27,20 +28,19 @@ supported:
 - HTTP
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
-
-In order to specify an external <abbr>dependency</abbr> for your stage, use the
-usual `-d` option in `dvc run` with the external path or URL to your desired
-file or directory.
+> Note [remote storage](/doc/command-reference/remote) is a separate feature.
 
 ## Examples
 
-Let's take a look at a `download_file` [stage](/doc/command-reference/run) that
-simply downloads a file from an external location.
+To define an external <abbr>dependency</abbr>, add the external URL or path to
+the `deps` field of `dvc.yaml`. For example, with the usual `-d` option in
+`dvc run`, giving it the external URL/path to your desired file or directory.
+
+Let's take a look at defining and running a `download_file` stage that simply
+downloads a file from an external location, on all the supported location types.
 
-> Note that some of these commands use the `/home/shared` directory, typical in
-> Linux distributions.
+> Note that some of the example commands below use the `/home/shared` directory,
```

Comment:
this note is also not needed here I think

Reply:
Gone.